R/data_reverse.R

```diff
@@ -2,8 +2,14 @@
 #'
 #' Reverse-score variables (change the keying/scoring direction).
 #'
-#' @param range Initial (old) range of values. If `NULL`, will take the range of
-#'   the input vector (`range(x)`).
+#' @param range Initial (old) range of values. For numeric variables, can be
```

Comment:
> "Initial (old) range of values"

I didn't change because it was the former wording, I just added some sentences. But we can edit this description completely to be clearer.

Reply:
I think we should edit it in this PR